### DISCUSSION TITLE
[Issue]

### DISCUSSION THREAD

**Discussion by bondhondas** [projectdiscovery/subfinder#721] [Q&A] [answered]
[FTL] Could not read configuration file /home/kali/.config/subfinder/config.yaml: EOF
![Screenshot 2022-11-16 051850](https://user-images.githubusercontent.com/19687230/202027463-dafefee1-c8a8-4c66-93b1-e3bae6a7f364.jpg)
![Screenshot 2022-11-16 051850](https://user-images.githubusercontent.com/19687230/202027508-3de18e0c-4758-4425-ab75-7712c7ee21dd.jpg)

**Comment by ehsandeep** [comment, marked as answer]
it's same issue as https://github.com/projectdiscovery/nuclei/issues/2853, permission issue on system to write config files.
